src/XamlMath.Shared/TexFormulaParser.cs

```diff
@@ -296,6 +293,20 @@ internal TexFormula Parse(SourceSpan value, string? textStyle, ICommandEnvironme
             return formula;
         }
 
+        private void ProcessLeftGroupChar(SourceSpan value, ref int position, string textStyle, ICommandEnvironment environment, TexFormula formula, int initialPosition)
```

Comment:
```suggestion
        private void ProcessLeftGroupChar(SourceSpan value, ref int position, string? textStyle, ICommandEnvironment environment, TexFormula formula, int initialPosition)
```

Remember the compiler warnings and that `textStyle` is nullable.